apps/web/components/dashboard/modal.tsx

```diff
@@ -0,0 +1,73 @@
+import ReactModal from "react-modal";
```

Comment:
What is the purpose of this that is not already covered by shadcn's Dialog?

Reply:
Yeah I mean radix dialog is fine I just think react-modal has more accessibility support. I can change it to shadcn's if u insist tho.

Reply:
Please use shadcn as we already using it

Reply:
done